app/lib/tool/test_profile/importer.dart

```diff
@@ -75,10 +75,13 @@ Future<void> importProfile({
   }
 
   // create versions
-  for (final testPackage in profile.packages) {
+  final orderedPackages =
+      _sortByShaHash<TestPackage>(profile.packages, (tp) => tp.name);
+  for (final testPackage in orderedPackages) {
     final packageName = testPackage.name;
     User lastActiveUser;
-    for (final versionName in testPackage.versions) {
+    final orderedVersions = _sortByShaHash(testPackage.versions);
```

Comment:
Do we need to create versions in arbitrary order? that seems weird.

Wouldn't those normally be created in the natural order? Is there any reason, not to create them in the order they appear in the test-profile?